packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -366,7 +366,7 @@ const DispatcherProxyHandler = {
     const error = new Error('Missing method in Dispatcher: ' + prop);
     // Note: This error name needs to stay in sync with react-devtools-shared
     // TODO: refactor this if we ever combine the devtools and debug tools packages
-    error.name = 'UnsupportedFeatureError';
+    error.name = 'ReactDebugToolsUnsupportedFeatureError';
```

Comment:
How about `ReactDebugToolsError` error, with the message being "Unsupported feature"?

Reply:
Then I would need to always check the message to decide what to show in different scenarios. Since we are not using custom error classes (`instanceof` not possible), I'd prefer to have a reliable way to detect what exactly it is. Checking `name` is OK but checking `message` will be too fragile.

Reply:
IMO checking the `name` does seem slightly better, but both are just strings (and string values like this are kind of a gray area in terms of semver API compatibility).

If our goal is to only check `name` though, "UnsupportedFeature" seems a bit vague. (Is it safe to assume that type of error only ever refers to a hook API we don't yet support? If not do we need to _also_ check the `message` to narrow things down?)

Reply:
OK I'll go with `ReactDebugToolsUnsupportedHookError`